src/http.cc

```diff
@@ -1926,10 +1926,10 @@ HttpStateData::httpBuildRequestHeader(HttpRequest * request,
         delete cc;
     }
 
-    /* maybe append Connection: keep-alive */
-    if (flags.keepalive) {
-        hdr_out->putStr(Http::HdrType::CONNECTION, "keep-alive");
-    }
+    // Explicitly enable persistent connections with legacy HTTP/1.0 servers
```

Comment:
I find the wording misleading: We often do not know (and do not check even when we could know) which server version we are talking to. I suggest: `Always send Connection because older servers need explicit "keep-alive" while modern servers need explicit "close", and we do not always know the server expectations.`

Reply:
Ok.